.github/workflows/kube-linter.yaml

```diff
@@ -0,0 +1,59 @@
+name: Check Kubernetes YAMLs with kube-linter
+
+on:
+  # Note that both `push` and `pull_request` triggers should be present for GitHub to consistently present kube-linter
+  # SARIF reports.
+  pull_request:
+    branches: [ main ]
+jobs:
+  scan:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      # This prepares directory where github/codeql-action/upload-sarif@v1 looks up report files by default.
+      - name: Create ../results directory for SARIF report files
+        shell: bash
+        run: mkdir -p ../results
+
+      - name: Setup Kustomize
+        uses: multani/action-setup-kustomize@v1
+        with:
+          version: 5.1.1
+
+      - name: Verify if the manifests were built
+        run: |
+            rm -rf auto-generated
+            ./build-manifests.sh
```

Comment:
Where is this file coming from? Does it suppose to be inside our repo?

Reply:
it s early to add kustomize build to run the kubelinter. will remove it for now